src/test/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/helper/builders/DSL.kt

```diff
@@ -336,32 +342,6 @@ interface CourseScheduleDsl
 @TestDataDslMarker
 interface CourseScheduleRuleDsl
 
-@TestDataDslMarker
-interface CourseAllocationDsl {
-  @CourseAllocationPayBandDslMarker
-  fun payBand(
-    startDate: String = "2022-10-31",
-    endDate: String? = null,
-    payBandCode: String = "5",
-  )
-
-  @CourseAttendanceDslMarker
-  fun courseAttendance(
-    courseSchedule: CourseSchedule,
-    eventId: Long = 0,
-    eventStatusCode: String = "SCH",
-    toInternalLocationId: Long? = -8,
-    outcomeReasonCode: String? = null,
-    paidTransactionId: Long? = null,
-  )
-}
-
-@TestDataDslMarker
-interface CourseAllocationPayBandDsl
-
-@TestDataDslMarker
-interface CourseAttendanceDsl
```

Comment:
Moved into entity based files that contain the DSL interfaces, the entity's dedicated Repository and the entity's builder factory.